### ISSUE TITLE
In the contact us page  Events page button  not functioning

### ISSUE THREAD

**Issue by DaSeeker67** [NITRR-Open-Source-Community/NOSC-Website#103]
in the contact us page  Events page button  not functioning as it throughs error message 

Easy difficulty

1. Go to 'nosc.netlify.app'
2. Click on  contact us button in the navigation bar
3. now click on Event  button in the navigation bar

**Expected behavior**
the user should be redirected to events page from contact us page 

**Screenshots**
![WhatsApp Image 2023-10-09 at 23 24 41](https://github.com/NITRR-Open-Source-Community/NOSC-Website/assets/65525769/7f123b99-934c-4033-91c6-dede97383af5)
![WhatsApp Image 2023-10-09 at 23 24 46](https://github.com/NITRR-Open-Source-Community/NOSC-Website/assets/65525769/2153df5f-3fff-45d8-9d99-93399becd83b)

**Comment by mzrokz** [comment]
I would like to pick up this issue, could this be assigned to me please.

**Comment by HamnaJabbar** [comment]
hi @DaSeeker67 !will you plz assign this to me? thankyou

**Comment by himanchal08** [comment]
i would like to work on this issue it seems doable ,i will try my best to implement it

**Comment by DaSeeker67** [comment]
> i would like to work on this issue it seems doable ,i will try my best to implement it

hello @himanchal08, nice to know you are eager to collaborate but in this repo we follow first come first serve basis , you will get many good first issues soon don't worry just star this repo and turn on the issue notification , hope you understand.

**Comment by mzrokz** [comment]
@DaSeeker67 I am having trouble reproducing the issue. I have tried in chrome and brave at the moment.
